docs/controls/LottieAnimationView.md

```diff
@@ -0,0 +1,150 @@
+---
+title: LottieAnimationView XAML Control
+author: azchohfi
+description: The LottieAnimationView allows you to render an Adobe After Effects animation in your UWP App
+keywords: windows 10, uwp, uwp community toolkit, uwp toolkit, lottie, lottieuwp, animations, aftereffects, XAML Control, xaml
+---
+
+# LottieAnimationView XAML Control
+
+The **LottieAnimationView Control** allows you to render an Adobe After Effects animation in your UWP App. It lets you control the Progress, the Scale, the Frame Rate, the Speed, it's behaviour for loops and the start and ending frames.
+
+## How it works?
+Lottie loads Json files that are exported from Adobe After Effects using a plugin called [Bodymovin](http://airbnb.io/lottie/after-effects/bodymovin-installation.html).
+
+It parses this Json, creating all the layers and animations in memory, and draws them using [Win2D](https://github.com/Microsoft/Win2D). These animations are completely unbound to the UI Thread, so they have an excelent performance. They are drawn depending on the frame rate of the animation, so if your animation runs at 30 FPS, it will only draw it 30 times per second.
+
+## LottieAnimationView Properties
```

Comment:
Should breakout in table like in latest doc sample template.

Reply:
Done.